modules/netstat/netstat.go

```diff
@@ -62,6 +63,7 @@ type element struct {
 	LocalPort     float64 `json:"localport,omitempty"`
 	RemoteAddr    string  `json:"remoteaddr,omitempty"`
 	RemotePort    float64 `json:"remoteport,omitempty"`
+	Namespace     string  `json:"namespace,omitempty"`
```

Comment:
It's intended to be a more abstract indicator. The identifier is uint on Linux, but the idea was this field could be utilized with other platforms if we run into it in the future. Resolved name of field.